app/views/WhatYouOwe.scala.html

```diff
@@ -375,6 +393,22 @@
             @messages("whatYouOwe.payments-made-bullet-1.3")</li>
         <li> @messages("whatYouOwe.payments-made-bullet-2")</li>
     </ul>
+
+    @if(creditAndRefundEnabled && whatYouOweChargesList.balanceDetails.availableCredit.isDefined && whatYouOweChargesList.balanceDetails.availableCredit.get > 0) {
+        <div class="govuk-body" id="money-in-your-account">
+        @if(isAgent){
```

Comment:
minor comment: should be space before left brace.

Reply:
Done.